### DISCUSSION TITLE
Energy dashboard - show the average temperature

### DISCUSSION THREAD

**Discussion by gilder0** [home-assistant/frontend#10796] [Dashboard]
It would be ideal to show the average temperature value in the energy consumption graphs, as the energy consumption (electricity, gas) for heating depends largely on it.

**Comment by nyroDev** [comment]
Moreover, if possible, it could be also useful to add weather data (raining, cloudy, etc...), as it's important when having solar panel that largely depends on it of course.  
Showing this data down to the hourly bar could be useful.

**Comment by xjikka** [comment]
I agree, it would be very useful. At least adding temperature - it would be enough to allow add temperature sensor on same place with other comsumption devices, and display it in the standard way.
If another sensor can be added, information about the percentage of precipitation or the percentage of cloud cover could be added in a similar way.

**Comment by karwosts** [comment]
I don't think there's any interest in adding more sensors to Energy right now, but you can always make your own [custom energy dashboard](https://www.home-assistant.io/dashboards/energy/) and add as much extra sensor information as you want to it.

**Comment by xjikka** [comment]
@karwosts I can create my own dashboard, but I couldn't find how to create my own custom energy dashboard even in doc. Can you please advise me? It is not possible add an energy card to a normal dashboard (or I don't know how).

**Reply by karwosts** [reply]
Create a card, select "Manual" type, and enter the card types in yaml as listed in my link above. 

e.g. `type: energy-usage-graph`

**Comment by xjikka** [comment]
@karwosts Wow, thank you, that's what I'm looking for. I didn't know that.
👍👍👍
